### PR TITLE
member profile page

### PR DESCRIPTION
**Pull request by waracci**
Draft, work ongoing. I suspect it is gonna be a massive PR

Some changes are propagating to other pages as well

Current: 
Screenshot from 2021-10-28 02-09-40

With new changes:
Screenshot from 2021-10-28 02-09-55

### CODE REVIEW

File: pombola/templates/default_base.html [hunk 1]
```diff
@@ -136,13 +138,20 @@
...
-          <script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
-          <script src="//www.google.com/jsapi"></script>
-          {% if settings.GOOGLE_RECAPTCHA_SITE_KEY %}
-              <script src="https://www.google.com/recaptcha/api.js" async defer></script>
-          {% endif %}
-          {% javascript 'base' %}
-          {% javascript 'survey' %}
+
+
+        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
```
Comment (jbothma) [inline pombola/templates/default_base.html:143]: if you rely on jquery 3.2.1 you'll have to remove this version and check that the site still works properly

File: pombola/templates/default_base.html [hunk 2]
```diff
@@ -9,6 +9,8 @@
 {% endcomment %}
 {% block ga_experiment %}
 {% endblock %}
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```
Comment (jbothma) [inline pombola/templates/default_base.html]: move to the rest of the CSS link tags

File: pombola/templates/default_base.html (hunk 1 above)
Comment (waracci) [reply pombola/templates/default_base.html:143]: A lot of errors happen when I remove the previous Jquery

Screenshot from 2021-11-03 18-09-53

File: pombola/templates/default_base.html (hunk 2 above)
Comment (waracci) [reply pombola/templates/default_base.html]: Done. Also, I discovered the ordering matters. If I move it any lower, it alters the page CSS significantly.

**Pull request merged**